tests/unit/tests/Helpers/EnqueueScriptTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace unit\tests\Helpers;
+
+use Give\Helpers\EnqueueScript;
+use Give_Unit_Test_Case;
+
+class EnqueueScriptTest extends Give_Unit_Test_Case
+{
+    public function testScriptRegistration()
+    {
+        $script = $this->getEnqueueScriptClassObject();
```

Comment:
If this is called in each test, then we can move it to the `setUp()` method (be sure to also call the parent `setUp()` method) and reference the value as a property of the test case.

Reply:
fixed: https://github.com/impress-org/givewp/compare/9c2383b48bbf...1c5e58d09cb8